docs/design/jet/025-jet-job-member-pruning.md

```diff
@@ -0,0 +1,193 @@
+# Jet job member pruning
```

Comment:
title of this TDD is "member pruning", but it describes more that that. Update it to better reflect the content

Reply:
you kept "member pruning" which was wrong